crates/sui/src/client_commands.rs

```diff
@@ -1401,6 +1401,40 @@ impl Display for SuiClientCommandResult {
                 table.with(style);
                 write!(f, "{}", table)?
             }
+            SuiClientCommandResult::Gas(gases) => {
+                if gases.is_empty() {
+                    write!(f, "No gas coins are owned by this address")?;
+                    return Ok(());
+                }
+
+                let mut builder = TableBuilder::default();
+                builder.set_header(vec!["gasCoinId", "coinValue"]);
+                for coin in gases {
+                    builder.push_record(vec![
+                        coin.gas_coin_id.to_string(),
+                        coin.coin_value.to_string(),
+                    ]);
+                }
+                let mut table = builder.build();
+                table.with(TableStyle::rounded());
+                if gases.len() > 10 {
+                    table.with(TablePanel::header(format!(
+                        "Showing {} gas coins and their values.",
+                        gases.len()
+                    )));
+                    table.with(TablePanel::footer(format!(
+                        "Showing {} gas coins and their values.",
+                        gases.len()
+                    )));
+                    table.with(TableStyle::rounded().horizontals([
+                        HorizontalLine::new(1, TableStyle::modern().get_horizontal()),
+                        HorizontalLine::new(2, TableStyle::modern().get_horizontal()),
+                        HorizontalLine::new(gases.len() + 2, TableStyle::modern().get_horizontal()),
```

Comment:
what is the gases.len() + 2 for?

Reply:
This will draw a line between the last item in the table and the footer, similarly as it is done for the header. the +2 is due to the fact that there are two rows in the beginning: header text + header with column names. If you take a look at the screenshots, you can see these three horizontal lines that are defined there.